python/lsst/obs/subaru/objectMasks.py

```diff
@@ -0,0 +1,132 @@
+import re
+import lsst.daf.base as dafBase
+import lsst.pex.logging as pexLog
+import lsst.afw.coord as afwCoord
+import lsst.afw.geom as afwGeom
+import lsst.afw.table as afwTable
+
+class ObjectMaskCatalog(object):
+    """Class to support bright object masks
+
+    N.b. I/O is done by providing a readFits method which fools the (old) butler.
```

Comment:
This was Robert being cleaver adding comment assuming people would be reading this code after the new butler is already in